cypress/support/commands.js

```diff
@@ -0,0 +1,69 @@
+// ***********************************************
+// CYPRESS - Custom commands
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+// NOTE: Please be mindful of what you log, try disabling logs when possible, otherwise you spam the inspector
+
+Cypress.Commands.add('awaitXHR', () =>
```

Comment:
This command makes Cypress wait for any API calls before proceeding, preventing race conditions.